setup.cfg

```diff
@@ -18,7 +18,7 @@ classifiers =
 package_dir =
     = src
 packages = find:
-python_requires = >=3.10
+python_requires = <=3.12
 install_requires =
     watchgod==0.7
```

Comment:
You should update the install_requires as well. `python-minifier` and `watchgod` was afaik only used by `flare.py`.